popup.css

```diff
@@ -0,0 +1,319 @@
+#popup-modal {
+  display: none;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: rgba(235, 236, 240, 0.941);
+  width: 100%;
+  height: 100vh;
+  z-index: 10;
+  padding: 0;
+  overflow: auto;
+}
+
```

Comment:
- [ ] Great job implementing the popup :+1: , But just as suggested by the previous reviewer, I think there are still some improvements you need to make before moving to the next milestone.
- [ ] This section of the popup does not look like the Figma design, Please try to restyle this as you are meant to get your portfolio to look exactly like the design.

- Your Project:
![pp-up](https://user-images.githubusercontent.com/107632780/233811953-4f7e3a02-772e-4257-a798-d6be1d053cc6.png)

- Figma design:
![Screenshot from 2023-04-23 00-28-19](https://user-images.githubusercontent.com/107632780/233811963-c0e3d790-b58a-465d-b68b-1fe789dca8b5.png)